feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -138,10 +146,14 @@ internal fun ForYouScreen(
     onboardingUiState: OnboardingUiState,
     feedState: NewsFeedUiState,
     deepLinkedUserNewsResource: UserNewsResource?,
+    askShowDialog: Boolean,
+    askNotificationPermission: Boolean,
     onTopicCheckedChanged: (String, Boolean) -> Unit,
     onTopicClick: (String) -> Unit,
     onDeepLinkOpened: (String) -> Unit,
     saveFollowedTopics: () -> Unit,
+    showNotificationPermissionDialog: () -> Unit,
+    showNotificationPermission: () -> Unit,
```

Comment:
I suggest renaming these to `onNotificationPermissionDismiss` and `onNotificationPermissionConfirm` alternatively